ddtrace/appsec/_iast/__init__.py

```diff
@@ -59,7 +60,7 @@ def ddtrace_iast_flask_patch():
     module_name = inspect.currentframe().f_back.f_globals["__name__"]
     module = sys.modules[module_name]
     try:
-        module_path, patched_ast = astpatch_module(module, remove_flask_run=True)
+        module_path, patched_ast = astpatch_module(module, remove_flask_run=False)
```

Comment:
If `remove_flask_run` is not needed anymore (I think it was only used here as `True`), we can remove the parameter.

Reply:
removed.